src/component/canvas/ScatterRenderable.jsx

```diff
@@ -111,6 +111,8 @@ export class ScatterRenderable extends React.PureComponent {
         index: number
     ): ?React.Element<any> {
         if(row.x == null || row.y == null) return null;
+        if(isNaN(row.x) || isNaN(row.y)) return null;
```

Comment:
It's pretty nitpicky but these two if statements could probably be in one seeing as it is returning `null` if any of the conditions are true. I was also wondering if it should be `if(typeof row.x !== 'number' || typeof row.y !== 'number')` to cover all bases, I don't think it would render correctly if anything other than numbers were passed in?